react/react1/week1/static-todo-list/src/App.js

```diff
@@ -0,0 +1,8 @@
+// import './App.css';
```

Comment:
Don't leave a commented-out code. In a real-world your colleagues would not understand why it's still there even if you know why you kept it. The other scenario can be if you write a comment above explaining the reason for keeping it.